### DISCUSSION TITLE
Welcome to spinorama Discussions!

### DISCUSSION THREAD

**Discussion by pierreaubert** [pierreaubert/spinorama#236] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by totti1965** [comment]
Hi,

I‘m Thorsten from Bonn, Germany. Mostly I‘m interested in Loudspeakers.
The passive Spinorama.org top Performer was the Epos ES 14 N by Karl-Heinz Fink with a rating (with equalisation) of 7.4

The data were taken by the private blog of Karl-Heinz. And taken with the Klippel NFS in the Style of Amir‘s measurements.

I found a data sheet of the FinkTeam WM4 also.

Would it be possible to incorporate the FinkTeam WM 4 Data in your Spinorama Database also?

link:

https://www.finkteam.com/wp-content/uploads/2020/10/Download_FinkAudio-WM-4-WP_issue2.5.pdf


Yours,

Thorsten

**Reply by pierreaubert** [reply]
Hi Thorsten,

There is not information to plot a spinorama in this document. The speaker looks great (like all Fink team speakers). You can ask them the full spinorama, I am sure they have it. If they are open to publish them, they can reach out and I will publish them.

Pierre

**Comment by totti1965** [comment]
Hi Pièrre,Thank you, Pièrre, for your quick answer! And thank you for all the work you put into the Spinorama website!The world would be so poor without people like Sean Olive, Amir, Erin, Crinacle or you!I tell as much people as I can about the existence of audiosciencereview.com and Spinorama.org in the German HiFi Forums where I have signed up!It was in February 2023 that I first hear of the existence of those websites!So I am a Late Beginner - but let’s fight together against Snakeoil and HiFi-Voodoo!Your big fan,ThorstenVon meinem iPad gesendetAm 05.11.2023 um 13:52 schrieb Pierre Aubert ***@***.***>:﻿
Hi Thorsten,
There is not information to plot a spinorama in this document. The speaker looks great (like all Fink team speakers). You can ask them the full spinorama, I am sure they have it. If they are open to publish them, they can reach out and I will publish them.
Pierre

**Comment by totti1965** [comment]
Hi Pierre,I wrote an E-Mail to the makers of the new Nubert Active-Loudspeaker Series……Hope, they will answer!Hello Mr. Bien, hello Mr. Pedal,For the European Nubert fans at audiosciencereview.com, could you possibly send the frequency responses of the new NuZeo speakers measured with the Klippel NFS to Pierre Aubert at www.Spinorama.org?I'm sure that all of the new NuZeo alto speakers are in the top 2% of the 800+ speakers currently in the database. Pierre Aubert has developed an algorithm that generates four different numerical values from the Klippel data of the speakers, which evaluate the behavior of the speakers in the rooma) Frequency response in the room (preference evaluation according to Sean Olive)b) Frequency response in the room with automatic 9-band parametric equalization;c) Frequency response in the room with 2 virtual, ideal-phase subwoofers with 14 Hz cut-off frequency; d) Frequency response in the room with the same equalizing + ideal subwooferPierre would be happy to add the Nubert speakers, including older models, to the database.By the way, about 65% of audiosciencereview.com forum members and Spinorama.org users are not North Americans! The readers are, just like Mr. Nubert (I still remember the photo with the tangled power cords, where he rightly pointed out that power cords simply don't make a sound difference), simply people who have something against hi-fi voodoo and "snake-oil" profiteering - and Nubert loudspeakers ultimately hit exactly this notch, since some of their products for EUR 2. 000,- to EUR 10.000,- pair price make many high-end products in the EUR 50.000,- to EUR 100.000,- pair price class look pretty old in terms of measurement and sound.Yours,Thorstenthw

**Comment by totti1965** [comment]
Bonjour Pierre,

Erin has testet the KEF R5 Meta on January 18.

His measurements are far better quality than the Kef mesurements.
Will you change / switch the KEF Data to Erins high quality Data?

Yours,

thorsten
Von meinem iPad gesendet

**Comment by pierreaubert** [comment]
Yes I will when I get them. KEF measurements are not bad at all btw.Sent from my iPhoneOn 4 Feb 2024, at 14:42, totti1965 ***@***.***> wrote:﻿
Bonjour Pierre,

Erin has testet the KEF R5 Meta on January 18.

His measurements are far better quality than the Kef mesurements.
Will you change / switch the KEF Data to Erins high quality Data?

Yours,

thorsten
Von meinem iPad gesendet

**Comment by pierreaubert** [comment]
I am currently uploading them.

[image: newplot.png]

On Sun, Feb 4, 2024 at 2:47 PM Pierre Aubert ***@***.***>
wrote:

> Yes I will when I get them. KEF measurements are not bad at all btw.
>
> Sent from my iPhone
>
> On 4 Feb 2024, at 14:42, totti1965 ***@***.***> wrote:
>
> ﻿
>
> Bonjour Pierre,
>
> Erin has testet the KEF R5 Meta on January 18.
>
> His measurements are far better quality than the Kef mesurements.
> Will you change / switch the KEF Data to Erins high quality Data?
>
> Yours,
>
> thorsten
> Von meinem iPad gesendet
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pierreaubert/spinorama/discussions/236#discussioncomment-8360756>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANBKHYBGTTV3YSQWBVQGUDYR6F53AVCNFSM6AAAAAAWTUVM22VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DGNRQG42TM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
>

-- 
--
PfA

**Comment by totti1965** [comment]
Bonjour Pierre,

Thank you for adding Erins Data for the KEF R5 Meta! As expected, the Data
are about 0.3 lower than the KEF Measurements.
So we can see, in which ballpark the other KEF Loudspekers might be, where
we just have the KEF Measurements: The difference might be about 0.3

Yours,

Thorsten

**Comment by totti1965** [comment]
Bonjour Pierre,

Il doit y avoir un problème avec les données d'Erins pour le Kef Blade 2
Meta !
Jetez un coup d'oeil aux données corrigées !
La correction rend la réponse en fréquence bien pire que sans correction :
Il n'y a absolument aucune atténuation des basses fréquences aux hautes
fréquences avec la réponse dans la pièce (normalement 4 dB ou 5 dB environ
de 50 Hz à 20 kHz !).

Il y a un problème avec la correction suggérée (estimation de la réponse
dans la pièce avec correction) !
C'est dommage, parce que c'est la meilleure enceinte passive dans la base
de données de spinorama.org jusqu'à présent !

https://www.spinorama.org/speakers/KEF%20Blade%202%20Meta/ErinsAudioCorner/index_eac.html

L'erreur ne se produit pas avec les données KEF!

Mais les données d'Erins Audio Corner sont normalement de meilleure qualité
que celles de l'entreprise!

A vous,
Thorsten

Traduit avec DeepL.com (version gratuite)

**Comment by pierreaubert** [comment]
Hi Thorsten,

I have updated the EQ and it should be better. You can see the result on
the GH thread.

Pierre

On Mon, Apr 29, 2024 at 7:59 PM totti1965 ***@***.***> wrote:

> Bonjour Pierre,
>
> Il doit y avoir un problème avec les données d'Erins pour le Kef Blade 2
> Meta !
> Jetez un coup d'oeil aux données corrigées !
> La correction rend la réponse en fréquence bien pire que sans correction :
> Il n'y a absolument aucune atténuation des basses fréquences aux hautes
> fréquences avec la réponse dans la pièce (normalement 4 dB ou 5 dB environ
> de 50 Hz à 20 kHz !).
>
> Il y a un problème avec la correction suggérée (estimation de la réponse
> dans la pièce avec correction) !
> C'est dommage, parce que c'est la meilleure enceinte passive dans la base
> de données de spinorama.org jusqu'à présent !
>
>
> https://www.spinorama.org/speakers/KEF%20Blade%202%20Meta/ErinsAudioCorner/index_eac.html
>
> L'erreur ne se produit pas avec les données KEF!
>
> Mais les données d'Erins Audio Corner sont normalement de meilleure
> qualité
> que celles de l'entreprise!
>
> A vous,
> Thorsten
>
> Traduit avec DeepL.com (version gratuite)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pierreaubert/spinorama/discussions/236#discussioncomment-9266321>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANBKH7TSTS473SKZ3UTJE3Y72DAVAVCNFSM6AAAAAAWTUVM22VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TENRWGMZDC>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
--
PfA

**Comment by denzel200219** [comment]
Hello. In the ASR review of the Neumann KH150, it received a normal score of 6.8 and a score w/sub of 8.6, yet in the SPINorama it shows the 8.6 in the w/EQ category, and a 7.3 in the w/sub category. Am I mistaken or are the w/EQ and w/SUB scores accidentally flipped in the SPINorama?

**Reply by totti1965** [reply]
Hello,
Everything seems to be normal on the Spinorama.org site:

The KH 150 is ranked

6.81 normal without correction
7.15 with EQ
8.55 with perfect 14 Hz Subwoofer
8.73 with perfect 14 Hz Subwoofer plus Equalizer

There were minor corrections of the data - but it was in February and just around 0.03 points.
I remember, that the Ascend Acoustics Sierra LX was (e. g.) lifted from 7.18 with EQ to 7.21 with EQ.
But I don’t know if there were any corrections for the Neuman KH 150.

Yours,

thorsten

**Comment by totti1965** [comment]
Hello,Everything seems to be normal on the Spinorama.org site:The KH 150 is ranked6.81 normal without correction7.15 with EQ8.55 with perfect 14 Hz Subwoofer8.73 with perfect 14 Hz Subwoofer plus EqualizerThere were minor corrections of the data - but it was in February and just around 0.03 points.I remember, that the Ascend Acoustics Sierra LX was (e. g.) lifted from 7.18 with EQ to 7.21 with EQ.But I don’t know if there were any corrections for the Neuman KH 150.Yours,thorstenVon meinem iPad gesendetAm 31.05.2024 um 06:24 schrieb denzel200219 ***@***.***>:﻿
Hello. In the ASR review of the Neumann KH150, it received a normal score of 6.8 and a score w/sub of 8.6, yet in the SPINorama it shows the 8.6 in the w/EQ category, and a 7.3 in the w/sub category. Am I mistaken or are the w/EQ and w/SUB scores accidentally flipped in the SPINorama?

**Reply by denzel200219** [reply]
![Image 5-31-24 at 1 19 PM](https://github.com/pierreaubert/spinorama/assets/171311264/125bcf20-f412-4744-87db-ac0e304f7b6c)

I see. But in the ranking, it shows the KH150 w/sub as 7.3, not 8.6/8.7 like you stated. I was just wondering why this is

**Comment by totti1965** [comment]
Oh, Yes! You are right. The complete column of the table is wrong. For all speakers.I am sure, Pierre will fix this soon.Thanks.Yours,thorstenVon meinem iPad gesendetAm 31.05.2024 um 19:23 schrieb denzel200219 ***@***.***>:﻿
Image.5-31-24.at.1.19.PM.jpg (view on web)
I see. But in the ranking, it shows the KH150 w/sub as 7.3, not 8.6/8.7 like you stated. I was just wondering why this is

**Comment by pierreaubert** [comment]
found and fixed! thanks.
Code will be pushed later today.

Pierre

On Sat, Jun 1, 2024 at 12:00 AM totti1965 ***@***.***> wrote:

> Oh, Yes! You are right. The complete column of the table is wrong. For all
> speakers.I am sure, Pierre will fix this soon.Thanks.Yours,thorstenVon
> meinem iPad gesendetAm 31.05.2024 um 19:23 schrieb denzel200219
> ***@***.***>:﻿
> Image.5-31-24.at.1.19.PM.jpg (view on web)
> I see. But in the ranking, it shows the KH150 w/sub as 7.3, not 8.6/8.7
> like you stated. I was just wondering why this is
>
> —Reply to this email directly, view it on GitHub, or unsubscribe.You are
> receiving this because you commented.Message ID: ***@***.***>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pierreaubert/spinorama/discussions/236#discussioncomment-9624031>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANBKHYFXAGNJJDXRUSMTZDZFDXGDAVCNFSM6AAAAAAWTUVM22VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TMMRUGAZTC>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
--
PfA
